execution/wasm/contract.go

```diff
@@ -72,6 +73,7 @@ type context struct {
 	code       []byte
 	output     []byte
 	returnData []byte
+	sequence   uint64
```

Comment:
Is the sequence populated from the account on execution?

Reply:
It was taken from execution sequence from EVM, 
```
func (vm *EVM) SetNonce(nonce []byte) {
	vm.options.Nonce = nonce
	vm.sequence = 0
}
```
since Nonce is also provided through initial wasm call, we can include account sequence there.